react_demo/cpp/index.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>FROM_C_PLUS_PLUS</title>
```

Comment:
This placeholder is not the only thing passed from C++, so it should be `PAGE_TITLE_FROM_C_PLUS_PLUS`.